server/routes/place/api.py

```diff
@@ -566,7 +566,7 @@ def populate_additional_category_data(category):
   all_places = [dcid]
   for t in BAR_CHART_TYPES:
     all_places.extend(raw_page_data.get(t + 'Places', []))
-  names = place_api.get_display_name('^'.join(sorted(all_places)), g.locale)
```

Comment:
This is intentionally dropped?

Reply:
Yes, the function is simplified by only taking a list of dcids.